hack/genpkgs/main.go

```diff
@@ -4,26 +4,23 @@ import (
 	"encoding/csv"
 	"fmt"
 	"log"
-	"math/rand"
 	"os"
 	"strconv"
-	"time"
 
 	"github.com/google/uuid"
 )
 
 const (
 	// How many records we want to generate.
-	datasetSize = 2500000
+	datasetSize = 55
```

Comment:
It should be fast, right? I believe it was initially created to test pagination performance, so maybe keep the default to 10000 or so.

Reply:
I only needed 41+ packages for my use case, but I will change the value to 10K. ;-)  I thought about adding a bash script + go script parameter to set the number of records generated, but decided not to go down that rabbit hole. :laughing: